stubs/Reflection.phpstub

```diff
@@ -67,10 +430,23 @@ class ReflectionFunction implements Reflector
     public function getAttributes(?string $name = null, int $flags = 0): array {}
 }
 
+class ReflectionFunction extends ReflectionFunctionAbstract
+{
+    /** @psalm-pure */
+    public function __construct(callable $function) {}
+
+    /**
+     * @return non-empty-string
+     *
+     * @psalm-pure
+     */
+    public function __toString(): string {}
+}
+
 class ReflectionProperty implements Reflector
 {
     /**
-     * @var string
+     * @var non-empty-string
```

Comment:
Property name can actually be an empty string: https://3v4l.org/K3kpQ

Reply:
I didn't see this: I will keep doing as if I didn't 😱

Reply:
d5cccbade209819c3a350ec96d89150bcb7bc7bb

Reply:
(commit message humorous, please do tell if you dislike :D )